lib/logflare/cluster/postgres_strategy.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Logflare.Cluster.PostgresStrategy do
```

Comment:
Maybe we could extract that and publish as a separate library? I think that quite a few people may find it useful.

Reply:
We definitely should.  Realtime and Supavisor are using it.

Reply:
task [here](https://www.notion.so/supabase/Logflare-Clustering-Extract-out-Postgres-Clustering-Strategy-to-its-own-separate-library-42d88bf841ad4338a19daac2aed11e25?pvs=4), backlogged